### DISCUSSION TITLE
How is the `_defualt.page.x.tsx` file to use chosen?

### DISCUSSION THREAD

**Discussion by ScottPierce** [vikejs/vike#1010] [Discussion] [answered]
I have two files:
* `../renderer/_default.page.server.tsx`
* `../../_default.page.server.tsx`

I assumed the renderer folder received some special handling, and that because it's closer, it'd be chosen, but it seems that the `../..` folder one is chosen, which is I suppose what I'd have originally expected.

1. How is the renderer folder treated when choosing the default page?
2. How is the default page to use selected when there are multiple to choose from?

**Comment by brillout** [comment, marked as answer]
`renderer/` comes last. Other than you're right: the FS nearest is chosen.

(FYI the order is defined internally by `getPageFilesSorter()`.)

I'll document this after the V1 design is released.
